docker-compose-test.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# docker-compose file for running tests in local
+#
+# Lint code using:
+#   docker-compose -f docker-compose-test.yaml up super-linter
+#
+version: "3.8"
+services:
+  # Basic development container.
+#  dev:
+#    # user: "${UID}:${GID}"
+#    image: gcr.io/distroless/java:11-debug-nonroot
+#    volumes:
+#      - .:/code
+#    entrypoint: [ "tail", "-f", "/etc/hosts" ]
```

Comment:
Shall we remove the bunch of commented lines? otherwise LGTM

Reply:
@KeerthanaR29 ping

Reply:
we need line 4 and 5 for command to execute that docker compose test.yaml..But we can remove other commands

Reply:
we need 10 - 15 lines.I forgot to uncomment that